tagcompare/capture.py

```diff
@@ -175,7 +225,8 @@ def main():
     output.aggregate()
 
     configs = settings.DEFAULT.configs_in_comparisons()
-    __capture_tags_for_configs(cids=cids, pathbuilder=pathbuilder, configs=configs)
+    _capture_tags_for_configs(
+        cids=cids, pathbuilder=pathbuilder, configs=configs)
```

Comment:
Good suggestion, I'll add a docstring for now.  Not sure if it's worth it at this point to change beyond that (certainly outside the scope of this PR)